### DISCUSSION TITLE
Guys I found a Zscaler bypass!

### DISCUSSION THREAD

**Discussion by Arthura0121** [3kh0/ext-remover#740] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
So I was just playing around with the settings and stuff until I found the Android Preferences app
I think we've all tried force stopping Zscaler but the wifi never worked after
Well, I found a solution to that
Steps:
(For those who know what this "Android Preferences" app I am talking about skip to step #8
1. Open your settings app and the category "Apps"
2. Click on where it says "Google Play Store"
3. Once you are on that page, you will see at the top something that says "Manage Android Preferences"
4. Click on that
5. Navigate to "Apps & Notifications" 
6. Click on "Apps Info"
7. Find the Zscaler app and click on it
8. Once you are there DON'T CLICK FORCE STOP YET!!!!
9. Go back to the other settings window and go to the category "Network"
10. There you will see a "VPN" option, click on that
11. Once you are on the VPN page, click on "Zscaler: Zscaler"
12. Look at the bottom and there you will see a "Proxy option", click that
13. You will now see a "Connection type" dropdown thing
14. Click on it and choose "Manual proxy configuration"
15. Once you do that go back to the "Android preferences" window and click on "Force Quit"
16. You are done! if nothing loads for you, you did the steps incorrectly and you have to do it again
Bypass made by @arthura0121 [ Me ;) ]

**Comment by 3kh0** [comment]
For our school that is now how it is managed, we are just using a simple proxy server. When LTBEEF was a thing, we could use that to disable the extension, which would unblock us from the filter

![image](https://github.com/3kh0/ext-remover/assets/58097612/14d9de2c-5544-40e9-92c3-33f1f9877ce4)
